.github/workflows/test-testgrid-results.yaml

```diff
@@ -0,0 +1,20 @@
+name: test-testgrid-results
+
+on:
+  schedule:
+    - cron: "0/15 * * * *"
```

Comment:
is this frequent enough? we are potentially delaying feedback for an additional 15 minutes